### PR TITLE
Add  frozen-util example

### PR DESCRIPTION
**Pull request by phamminh0811**
Closes: #4139 

## What is the purpose of the change
Adding frozen util example on create position cmd

## Testing and Verifying

This change is already covered by existing tests.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

File: x/concentrated-liquidity/client/cli/tx.go
Comment (mattverse) [inline x/concentrated-liquidity/client/cli/tx.go:36]:
```suggestion
		Use:                 "create-position [lower-tick] [upper-tick] [token-0] [token-1] [token-0-min-amount] [token-1-min-amount] [frozen-until]",
```

**Review by mattverse — COMMENTED** [review]
LGTM once merged with main to resolve current failing test and change requested has been resolved

**Review by p0mvn — APPROVED** [review]

**Pull request merged**
